src/components/common/panels/panels.module.scss

```diff
@@ -0,0 +1,57 @@
+@mixin generate-orders($max-index) {
+  @for $i from 0 to $max-index {
+    [data-columns='#{$i}'] {
+      grid-template-columns: repeat($i, minmax(0, 1fr));
+    }
+  }
+}
```

Comment:
make fixed columns

Reply:
They are fixed.

Reply:
Don't want to use scss features like this, it's ok when in some cases you repeat code, you can't get rid of repeating at all